blocks/resources/__logo-zone/resources__logo-zone.css

```diff
@@ -0,0 +1,6 @@
+.resources__logo-zone {
+    padding-top: 80px;
+    margin: auto;
```

Comment:
The `margin: auto;` property can behave unpredictably in this context, so it's better to use flex properties instead.